modules/thermal_hydraulics/test/tests/materials/ad_wall_heat_transfer_coefficient_kazimi/tests

```diff
@@ -0,0 +1,63 @@
+[Tests]
+  design = 'ADWallHeatTransferCoefficientKazimiMaterial.md'
+  issues = '#24340'
+  [normal_operation]
+    requirement = "The system shall compute the heat conductance using the Kazime-Carelli "
+                  "correlation for the Nusselt number"
+    type = 'CSVDiff'
+    input = 'kazimi_test.i'
+    csvdiff = 'kazimi_test_out.csv'
+  []
+  [abnormal_operation]
+    requirement = "The system shall return a warning message when: "
```

Comment:
The system shall return a warning message when the Kazimi-Carelli correlation is used with:

Reply:
done